inscriptions/ISic003534.xml

```diff
@@ -191,6 +196,13 @@
     </facsimile>
     <text>
         <body>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AL">
+                <ab>
+                    <lb n="1"/><persName type="attested"><name type="praenomen"><expan><abbr>M</abbr><ex>arcus</ex></expan></name><g type="interpunct">·</g><name type="nomen">Va<supplied reason="lost">lerius</supplied></name>                    ̣
+                        <lb n="2"/><orgName type="tribus" nymRef="#Pomptina"><expan><abbr>Pom</abbr><ex cert="low">ptina</ex></expan></orgName></persName> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
remove the nymRef; and the P of Pom should definitely be marked as `<unclear>` (Bitto is odd in her practices, and her description of the text/its reading is quite clear that there are some very faint traces which she thinks are compatible with P, but hardly more than that, and this is a clear case for underdotting, i.e. 'unclear'.)